### DISCUSSION TITLE
Zero-dimensional entity class

### DISCUSSION THREAD

**Discussion by Mastomaki** [Spine-project/Spine-Toolbox#2814] [Support]
What is zero-dimensional entity class? A normal object class would be one-dimensional?

![image](https://github.com/spine-tools/Spine-Toolbox/assets/42839563/b18ff24f-b978-4cfc-b7ec-1e550dd46d7c)

**Comment by soininen** [comment]
Zero-dimensional entity classes correspond to object classes in the legacy parlance. 1-dimensional entity classes are relationship classes with single object class dimension.

**Reply by Mastomaki** [reply]
1-dimensional entity classes select a subset of an underlying object class?

**Reply by soininen** [reply]
Yes.

**Reply by nnhjy** [reply]
Are we going to keep the two existences of the legacy object classes? They sometimes cause confusion.

**Reply by jkiviluo** [reply]
Where do we still use object classes?
